sunpy/coordinates/transformations.py

```diff
@@ -61,6 +63,48 @@
            'hme_to_gei', 'gei_to_hme', 'gei_to_gei']
 
 
+# Global variable to control debugging output
+_debug_output = False
```

Comment:
Why not get the logging level and use that as the default?

Reply:
If you mean replacing this line with something like `_debug_output = log.getEffectiveLevel() <= logging.DEBUG`, that would be evaluated on import, and I think that could be confusing.  In the example that I provided, it would evaluate as `False` because I don't set the logging level to "debug" until after the import.

Reply:
Hmm, it might be possible to defer the check of the logging level until the first time a transformation is used...

Reply:
Yeah checking it on execution of the transformation function was what I was thinking.

Reply:
Okay, it looks like the performance hit for checking the logging level with every transformation is negligible, so I'm doing that now.